bin.src/genDefectList.py

```diff
@@ -0,0 +1,265 @@
+#!/usr/bin/env python
+import argparse
+import lsst.afw.image as afwImage
+import lsst.afw.math as afwMath
+import lsst.daf.persistence as dafPersist
+from lsst.log import Log
+import numpy as np
+import os
+import pfs.drp.stella as drpStella
+try:
+    import lsst.afw.display as afwDisplay
+except ImportError:
+    afwDisplay = None
```

Comment:
I don't think so:
```
$ unsetup -j display_ds9
$ ipython
In [1]: import lsst.afw.display

In [2]: 
```

Can you check?  Are you setting an explicit ds9 backend somewhere?